docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+    Indeed, for certain subgroups in the data, accuracy may be improved dramatically.
+
+Fairlearn questions
+-------------------
+
+Can Fairlearn make my model fair?
+    Naive use of Fairlearn is not going to result in fairer models.
+    Indeed, when used incorrectly, Fairlearn could produce model which is *less* fair.
+
+    What Fairlearn can do is generate a model which has lower disparity as measured by
+    a user-specified metric.
+    Whether this makes the model fairer is outside the remit of Fairlearn - or indeed of
+    machine learning in general.
+    To determine whether the new model is fairer, the whole system of which it is a part
+    needs to be considered in terms of the societal context in which it is used.
+
+What sort of fairness related harms can Fairlearn address?
+    Fairlean concentrates on group fairness.
+    That is, obtaining the minimum disparity on some metric evaluated on
+    specified subgroups in the data.
+    This is certainly not the only way of looking at fairness, though.
+    For example, there is the concept of individual fairness (where decisions are evaluated
+    on the level of individuals rather than groups) and counter-factual fairness (e.g. does the
+    decision change if an individual's gender is changed from male to female).
+
+Can Fairlearn be used to detect bias in datasets?
+    We do not have concrete plans for this at the present time.
+
+Can Fairlearn recommend ways to make my model fairer?
+    No. Fairlearn is not magic.
+
+    Fairness is a social concept, and no technological solution can make
+    things fair automatically.
+    In order to use Fairlearn successfully, you first need to work out
+    what fairness means for your particular problem.
+    Once that is determined, you can then work out how to use the
+    various algorithms and fairness metrics which Fairlearn supports
+    to develop models which are *fairer by the standard you have chosen*.
+
+What unfairness mitigation techniques does Fairlearn support?
+    ExpGrad, PostProcessing....
+
+Which ML libraries does Fairlearn support?
+    We have generally followed conventions for `scikit-learn` in Fairlearn.
+    However, support is not restricted to Estimators from `scikit-learn`.
+    Any algorithm which provides (or can be wrapped to provide) `fit()` and
+    `predict()` methods should work.
+
+Does Fairlearn work for image and text data?
+    We have not (yet) looked at using Fairlearn on image or text data.
+    However, so long as the estimators used have `fit()` and `predict()` methods
+    as required by Fairlearn, it should be possible to use Fairlearn on
+    image or text models.
+
+Is Fairlearn available in languages other than Python?
+    For the moment, we only support Python >= 3.6
+
+Can I contribute to Fairlearn?
+    Absolutely! Please see our :ref:`Contributor Guide <contributor_guide>` for
+    information on how to add your contributions.
```

Comment:
I like this! Shall we also add "All are welcome." ?

Reply:
Maybe... I feel that might be repetitive, though.

Reply:
As someone who's read plenty of community pages I have not read a single "All are welcome" that I found repetitive :-)
Can't stress inclusion enough.

Reply:
Done. At the end so the repetition is less obvious.